package.json

```diff
@@ -32,13 +32,18 @@
     "async": "^2.5.0",
     "data-queue": "0.0.2",
     "debug": "^3.0.1",
+    "hapi": "^16.5.2",
```

Comment:
Can't we have those modules split up in client and server? Putting `hapi` and `socket.io` into this module only makes the client 10x bigger without any benefits for the user.

Reply:
Ok, we can do that then.

Reply:
Doing it https://github.com/libp2p/js-libp2p-websocket-star-rendezvous